src/vue/pages/Security.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="security-page">
+    <drawer :is-shown.sync="isDrawerShown">
+      <template v-if="viewMode === VIEW_MODES.enableTfa">
+        <template slot="heading">
+          <template v-if="isTotpEnabled">
+            {{ 'security-page.disable-tfa-title' | globalize }}
+          </template>
+          <template v-else>
+            {{ 'security-page.enable-tfa-title' | globalize }}
+          </template>
+        </template>
+        <tfa-form @update="updateTfa" />
+      </template>
+      <template v-else-if="viewMode === VIEW_MODES.changePassword">
+        <template slot="heading">
+          {{ 'security-page.change-password-btn' | globalize }}
+        </template>
+        <change-password-form />
+      </template>
+      <template v-else-if="viewMode === VIEW_MODES.viewAccountId">
+        <template slot="heading">
+          {{ 'security-page.account-id-title' | globalize }}
+        </template>
+        <key-viewer
+          :value="wallet.accountId"
+          :label="'security-page.account-address-label' | globalize"
+        />
+      </template>
+      <template v-else-if="viewMode === VIEW_MODES.viewSecretSeed">
+        <template slot="heading">
+          {{ 'security-page.secret-seed-title' | globalize }}
+        </template>
+        <div class="secret-seed">
+          <p class="secret-seed__description">
+            {{ 'security-page.secret-seed-desc' | globalize }}
+          </p>
+          <clipboard-field
+            :value="wallet.secretSeed"
+            :label="'security-page.secret-seed-title' | globalize"
+          />
+        </div>
+      </template>
+    </drawer>
+    <div class="security-page__row">
+      <p class="security-page__row-title">
+        {{ 'security-page.enable-tfa-title' | globalize }}
+      </p>
+      <div @click="showDrawer(VIEW_MODES.enableTfa)">
```

Comment:
do not use @click on non-tabable elements. only buttons and links allowed

Reply:
Fixed, use button